exasol/analytics/schema/column.py

```diff
@@ -31,7 +27,7 @@ def __get__(self, owner_self, owner_cls):
 
 
 @dataclass(frozen=True, repr=True, eq=True)
-class Column:
+class ColumnType:
```

Comment:
would SQLType maybe more correct?

Reply:
Not too bad in my eyes. I only already have a class [SqlType in column_type_utils.py](https://github.com/exasol/advanced-analytics-framework/blob/b6bcb6e2045e02709a439e9c7d4b13be612d9105/exasol/analytics/schema/column_type_utils.py#L65) already, that should be renamed then.